### PR TITLE
973 add plan and jurisdiction assignment map

### LINKED ISSUE #973: Plan Assignment: Add map
This needs to be done as a follow up on https://github.com/onaio/reveal-frontend/pull/929

### PR DESCRIPTION
**Pull request by KipSigei**
closes #973

### CODE REVIEW

File: src/containers/pages/AssigmentMapWrapper/index.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  getJurisdictionsById,
+  getJurisdictionsFC,
+  reducerName as jurisdictionReducerName,
+} from '../../../store/ducks/opensrp/jurisdictions';
+import { buildStructureLayers } from '../FocusInvestigation/map/active/helpers/utils';
+reducerRegistry.register(jurisdictionReducerName, jurisdictionReducer);
+
+/** props for Plan jurisdiction and team assignment higher order component */
+export interface AssignmentMapWrapperProps {
+  // plan: PlanDefinition;
```
Comment (p-netm) [inline src/containers/pages/AssigmentMapWrapper/index.tsx:30]: dead code

File: src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx [hunk 2]
```diff
@@ -234,7 +234,8 @@ export const buildJurisdictionLayers = (jurisdiction: Jurisdiction | null) => {
  * @param {FeatureCollection<StructureGeoJSON> | null} structs - the structures
  */
 export const buildStructureLayers = (
```
Comment (p-netm) [inline src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx:236]: please update the docstring

**Comment by p-netm** [comment]
Looks ok to me, will create an issue to add handlers and for drilling down and information to be shown during hover.

File: src/components/GisidaLite/index.tsx [hunk 3]
```diff
@@ -134,4 +125,19 @@ const GisidaLite = (props: GisidaLiteProps) => {
...
+ * @param prevProps
+ * @param nextProps
+ */
+export const arePropsEqual = (prevProps: GisidaLiteProps, nextProps: GisidaLiteProps) => {
+  if (!isEqual(prevProps.layers, nextProps.layers)) {
+    return false;
+  }
+
+  return true;
+};
```
Comment (ciremusyoka) [inline src/components/GisidaLite/index.tsx:139]: can we just `return isEqual(prevProps.layers, nextProps.layers)` here.

Comment (KipSigei) [reply src/components/GisidaLite/index.tsx:139]: @ciremusyoka addressed

File: src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx (hunk 2 above)
Comment (KipSigei) [reply src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx:237]: @p-netm addressed

**Pull request merged**